tests/unit/mesh/test_pmesh.cpp

```diff
@@ -118,6 +118,79 @@ TEST_CASE("ParMeshGlobalIndices",  "[Parallel], [ParMesh]")
    }
 }
 
+namespace simplicial
+{
+
+double exact(const Vector &xvec)
+{
+   // The exact solution is linear and is harmonic
+   return xvec[0] + xvec[1] + xvec[2];
+}
+
+void SolveDiffusionProblem(ParMesh &mesh, Vector &x_out)
+{
+   H1_FECollection fec(1, mesh.Dimension());
+   ParFiniteElementSpace fes(&mesh, &fec);
+
+   // Right-hand side is zero since exact solution is harmonic
+   ParLinearForm b(&fes);
+   b.Assemble();
+
+   ParBilinearForm a(&fes);
+   a.AddDomainIntegrator(new DiffusionIntegrator);
+   a.Assemble();
+
+   Array<int> ess_tdof_list, ess_bdr;
+   if (mesh.bdr_attributes.Size())
+   {
+      ess_bdr.SetSize(mesh.bdr_attributes.Max());
+      ess_bdr = 1;
+      fes.GetEssentialTrueDofs(ess_bdr, ess_tdof_list);
+   }
+
+   // Use the exact solution as boundary conditions
+   ParGridFunction x(&fes);
+   FunctionCoefficient exact_coeff(exact);
+   x.ProjectBdrCoefficient(exact_coeff, ess_bdr);
+
+   OperatorPtr A;
+   Vector B, X;
+   a.FormLinearSystem(ess_tdof_list, x, b, A, X, B);
+
+   CGSolver cg(MPI_COMM_WORLD);
+   cg.SetRelTol(1e-12);
+   cg.SetMaxIter(2000);
+   cg.SetPrintLevel(1);
+   cg.SetOperator(*A);
+   // X = 0.0;
+   cg.Mult(B, X);
+   x_out = X;
+}
+
+};
```

Comment:
Travis [doesn't like](https://travis-ci.com/github/mfem/mfem/jobs/482864299#L950) this semicolon.